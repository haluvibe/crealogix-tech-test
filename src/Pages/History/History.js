import React from 'react';
import { Link } from 'react-router-dom';

import CounterProvider from '../../Provider/CounterProvider';
import List from '../../Components/List/List';

const History = () => (
  <>
    <CounterProvider
    	render={({ history }) => (
    		<List
    			history={history}
    		/>
    	)}
    />
    <Link to="/">Counter</Link>
  </>
);

export default History;
