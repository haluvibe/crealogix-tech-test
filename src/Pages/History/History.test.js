import React from "react";
import { shallow } from "enzyme";
import History from "./History";

describe("History", () => {
  it("should render correctly with no props", () => {
    const wrapper = shallow(<History />);

    expect(wrapper).toMatchSnapshot();
  });
});
