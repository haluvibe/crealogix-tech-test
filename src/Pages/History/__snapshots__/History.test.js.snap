// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History should render correctly with no props 1`] = `
<Fragment>
  <Connect(CounterProvider)
    render={[Function]}
  />
  <Link
    replace={false}
    to="/"
  >
    Counter
  </Link>
</Fragment>
`;
